packages/flet_local_auth/lib/src/local_auth.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:io' show Platform;
+
+import 'package:flet/flet.dart';
+import 'package:flutter/widgets.dart';
+import 'package:local_auth/local_auth.dart';
+
+class LocalAuthenticationControl extends StatefulWidget {
+  final Control? parent;
+  final Control control;
+  final FletControlBackend backend;
+
+  const LocalAuthenticationControl(
+      {super.key,
+      required this.parent,
+      required this.control,
+      required this.backend});
+
+  @override
+  State<LocalAuthenticationControl> createState() =>
+      _LocalAuthenticationControlState();
+}
+
+class _LocalAuthenticationControlState
+    extends State<LocalAuthenticationControl> {
+  @override
+  Widget build(BuildContext context) {
+    debugPrint("LocalAuthentication build: ${widget.control.id}");
+
+    () async {
+      final LocalAuthentication auth = LocalAuthentication();
+
+      widget.backend.subscribeMethods(widget.control.id,
+          (methodName, args) async {
+        switch (methodName) {
+          case "available":
+            // get's available biometrics
+            final List<BiometricType> availableBiometrics =
+                await auth.getAvailableBiometrics();
+            // check's if device is supported
+            final bool canAuthenticate = availableBiometrics.isNotEmpty;
+            debugPrint("canAuthenticate: $canAuthenticate");
+            return "$canAuthenticate";
+
+          case "authenticate":
+            try {
+              if (Platform.isWindows) {
+                args['biometricsOnly'] = "false";
+              }
+              final bool didAuthenticate = await auth.authenticate(
+                // if no title is given, uses "Authentication required" as the title
+                localizedReason: args['title'] ?? "Authentication required",
+                options: AuthenticationOptions(
+                    biometricOnly:
+                        args['biometricsOnly'] == 'true' ? true : false,
+                    useErrorDialogs: true),
+              );
+              return "$didAuthenticate";
+            } on Exception catch (_) {
+              debugPrint("An error has occured: $_");
```

Comment:
The error message should be forwarded instead of ignored, don't you think?

Reply:
the error message is usually `user didn't authenticate` or `Authentication is not supported on this platform`. I can try making them a warning if that helps.